apps/web-mzima-client/src/app/settings/surveys/create-field-modal/create-field-modal.component.html

```diff
@@ -209,7 +209,10 @@
 
 <ng-template #allFields>
   <div class="fields-list">
-    <div
+    <mzima-client-button
+      color="default-color"
+      size="unspecified"
```

Comment:
@kivuvarosekivuvan I like your use of `default-color` for the color - can we do the same for the `size` for consistency sake, and I believe it's more understandable. 
Just change `unspecified` to `default-size`.

Reply:
Okay @Ifycode On it